src/features/locators/utils/utils.ts

```diff
@@ -14,14 +14,14 @@ import {
 } from "../types/locator.types";
 import { copyToClipboard, getLocatorString, getElementFullXpath } from "../../../common/utils/helpers";
 import { LocatorOption } from "./constants";
-import { getLocator } from "./locatorOutput";
 import { LocatorType } from "../../../common/types/common";
 import { isStringContainsNumbers } from "../../../common/utils/helpers";
 import { FormInstance } from "antd/es/form/Form";
 
-export const getLocatorWithJDIAnnotation = (locator: LocatorValue): string => `@UI("${locator.xPath}")`;
+export const getLocatorWithJDIAnnotation = (locator: string): string => `@UI("${locator}")`;
 
-export const getLocatorWithSelenium = (locator: LocatorValue): string => `@FindBy(xpath = "${locator.xPath}")`;
+export const getLocatorWithSelenium = (locator: string, option: LocatorOption): string =>
+  option.includes("xPath") ? `@FindBy(xpath = "${locator}")` : `@FindBy(css = "${locator}")`;
```

Comment:
I believe strict equality with `LocatorOption` can be used and will be better here,  instead of `includes`.
Moreover, the only place where condition is really needed is choosing xpath/css. You can write `@FindBy(${<your condition>} = "${locator}")`